### DISCUSSION TITLE
How to target specific node types in hetzner-k3s without node labels?

### DISCUSSION THREAD

**Discussion by abhishek-ch** [vitobotta/hetzner-k3s#499] [General]
I'm running a Kubernetes cluster with hetzner-k3s and have multiple node groups with different specifications. Since hetzner-k3s doesn't currently support node labels, I'm looking for alternative ways to schedule workloads on specific node types.

Currently, pods with high CPU/memory usage sometimes crash entire nodes. While I could overallocate resources, I'd prefer to properly direct workloads to appropriate node types.

Are there any workarounds or alternative methods to hint/guide pod scheduling to specific node types in this setup?

**Comment by vitobotta** [comment]
Node labels and taints are supported for static node pools at the moment but not for autoscaled ones. What kind of pools are you using for this?

**Reply by abhishek-ch** [reply]
Autoscaled one but it's good to know that it works on the static pod, then I can create static node pool with label
